assets/styles/style.scss

```diff
@@ -615,12 +615,16 @@ body.dark-theme {
 
 
 .email-container {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
 }
 
 .email-content {
     box-sizing: border-box;
     z-index: 1;
-    height: calc(100vh - #{$toolbarHeight});
+    height: 0;
```

Comment:
For clarity, `height` should be removed completely, instead of `height: 0` which is overridden by `flex-grow: 1`.